airbyte-integrations/connectors/source-okta/sample_files/catalog.json

```diff
@@ -0,0 +1,28 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "groups",
+        "json_schema": {}
+      },
+      "sync_mode": "incremental",
+      "destination_sync_mode": "overwrite"
+    },
+    {
+      "stream": {
+        "name": "logs",
+        "json_schema": {}
+      },
+      "sync_mode": "incremental",
```

Comment:
this catalog used for testing should contain the cursor field and primary key for each stream